packages/server/src/internals/graphql/utils.ts

```diff
@@ -105,8 +105,19 @@ export function decodeFromBase64(text: string) {
   return Buffer.from(text, 'base64').toString('utf8')
 }
 
-export function assertStringIsNotEmpty(...strings: string[]) {
-  strings.forEach((str) => {
-    if (!str) throw new UserInputError(`Argument ${str} may not be empty`)
-  })
+export function assertArgumentIsNotEmpty(args: { [key: string]: unknown }) {
+  const emptyArgs: string[] = []
+
+  for (const [key, value] of Object.entries(args)) {
+    if (!value) {
+      emptyArgs.push(key)
+    }
+  }
+  if (emptyArgs.length) {
```

Comment:
```suggestion
  if (emptyArgs.length > 0) {
```

It is easier to read, when the if conditions are more explicit